platforms/linux/build_model_check.sh 编译报错

OS： centos 7.2
GCC： <= 4.9.0 以下，会出现下面的编译报错：
```text
/data/lucas/git-hub/TNN/source/tnn/device/opencl/opencl_runtime.cc: 在成员函数‘tnn::Status tnn::OpenCLRuntime::Init()’中:
/data/lucas/git-hub/TNN/source/tnn/device/opencl/opencl_runtime.cc:158:108: 错误：对重载的‘Context(<brace-enclosed initializer list>, std::nullptr_t, std::nullptr_t, std::nullptr_t, cl_int*)’的调用有歧义
         context_ = std::shared_ptr<cl::Context>(new cl::Context({*device_}, nullptr, nullptr, nullptr, &err));
                                                                                                            ^
/data/lucas/git-hub/TNN/source/tnn/device/opencl/opencl_runtime.cc:158:108: 附注：备选是：
In file included from /data/lucas/git-hub/TNN/source/tnn/device/opencl/opencl_wrapper.h:32:0,
                 from /data/lucas/git-hub/TNN/source/tnn/device/opencl/opencl_runtime.h:24,
                 from /data/lucas/git-hub/TNN/source/tnn/device/opencl/opencl_runtime.cc:15:
/data/lucas/git-hub/TNN/source/tnn/device/opencl/../../../../third_party/opencl/include/CL/cl2.hpp:2674:5: 附注：cl::Context::Context(const cl::Device&, cl_context_properties*, void (*)(const char*, const void*, cl::size_type, void*), void*, cl_int*)
     Context(
     ^
/data/lucas/git-hub/TNN/source/tnn/device/opencl/../../../../third_party/opencl/include/CL/cl2.hpp:2643:5: 附注：cl::Context::Context(cl::vector<cl::Device>&, cl_context_properties*, void (*)(const char*, const void*, cl::size_type, void*), void*, cl_int*)
     Context(
     ^
在全局域：
cc1plus: 警告：无法识别的命令行选项“-Wno-ignored-attributes” [默认启用]
cc1plus: 警告：无法识别的命令行选项“-Wno-pass-failed” [默认启用]
make[2]: *** [source/tnn/device/opencl/CMakeFiles/TNNOpenCL.dir/opencl_runtime.cc.o] 错误 1
make[1]: *** [source/tnn/device/opencl/CMakeFiles/TNNOpenCL.dir/all] 错误 2
make: *** [all] 错误 2
```

--- a/source/tnn/device/opencl/opencl_runtime.cc
+++ b/source/tnn/device/opencl/opencl_runtime.cc
@@ -145,17 +145,17 @@ Status OpenCLRuntime::Init() {
         LOGE("eglGetCurrentContext(): 0x%x\n", eglGetCurrentContext());
         cl_context_properties context_prop[] = {CL_GL_CONTEXT_KHR, (cl_context_properties)eglGetCurrentContext(),
                                                 CL_EGL_DISPLAY_KHR, (cl_context_properties)eglGetCurrentDisplay(), 0};
-        context_ = std::shared_ptr<cl::Context>(new cl::Context({*device_}, context_prop, nullptr, nullptr, &err));
+        context_ = std::shared_ptr<cl::Context>(new cl::Context(*device_, context_prop, nullptr, nullptr, &err));
 
         if (err != CL_SUCCESS) {
             LOGE(
                 "Create special opencl context falied, Create common opencl "
                 "context then.\n");
-            context_ = std::shared_ptr<cl::Context>(new cl::Context({*device_}, nullptr, nullptr, nullptr, &err));
+            context_ = std::shared_ptr<cl::Context>(new cl::Context(*device_, nullptr, nullptr, nullptr, &err));
         }
 #else
         LOGI("Create common opencl context\n");
-        context_ = std::shared_ptr<cl::Context>(new cl::Context({*device_}, nullptr, nullptr, nullptr, &err));
+        context_ = std::shared_ptr<cl::Context>(new cl::Context(*device_, nullptr, nullptr, nullptr, &err));
 #endif
         if (err != CL_SUCCESS) {
             LOGE("Context create failed! (ERROR CODE: %d)\n", err);

